var/spack/repos/builtin/packages/opengl/package.py

```diff
@@ -31,11 +31,17 @@ class Opengl(Package):
     homepage = "https://www.opengl.org/"
     url      = "https://www.opengl.org/"
 
-    version('3.2', 'N/A')
+    # URL here is arbitrary, just so that spack's fetcher does not throw
```

Comment:
@goxberry can you add a reference to https://github.com/spack/spack/issues/7061 here?

Reply:
I've added a reference in the comment.

Reply:
@goxberry apologies but I just looked at this closer and realized that the URL is a real URL, so if I understand correctly this goes through all the effort of downloading. In that case could you do one of the following:

* (if you have the time): override the `Package.fetcher` property to raise the error that is currently printed during the install phase. In this case you should not have to define a url and this would avoid downloading something unnecessarily before failing (and also potentially running into other versioning/fetching related errors that would result from changes to the hosting of the tarball at the url).
* prefix this commit with "XXX:" like `XXX: URL here is arbitrary...`

The override would look like:

```
    version('3.2', 'N/A')

    @property
    def fetcher(self):
        raise InstallError("The same error message as for install()")
```

Let me know your opinion.